lib/backend/backend.go

```diff
@@ -105,7 +105,7 @@ type Batch interface {
 //
 // lease, err := backend.Create()
 // lease.Expires = time.Now().Add(time.Second)
-// // Item TTL is extended
+// Item TTL is extended
```

Comment:
Hmm, this duplicate `// ` might actually be valid, since that comment is part of a code example?